src/main/java/com/elastic/support/diagnostics/commands/ElasticVersionCheckCmd.java

```diff
@@ -20,7 +19,7 @@
 import java.util.Map;
 
 
-public class VersionCheckCmd implements Command {
+public class ElasticVersionCheckCmd implements Command {
```

Comment:
Seems odd that this is called `Elastic` instead of `Elasticsearch`.

Reply:
Just opted for shorter, but I don't have any particular attachment otherwise. Updated in 2nd commit.